lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -956,7 +973,148 @@ section.instructor-dashboard-content-2 {
       }
     }
   }
+
+  /*
```

Comment:
Sorry, this was already merged from _cohorts-management.scss, I just neglected to remove the comment.